bin/terminal_velocity

```diff
@@ -60,6 +61,11 @@ the default default will be used"""
         help="the filename extensions to recognize in the notes dir, a "
             "comma-separated list (default: %(default)s)")
 
+    parser.add_argument("--hooks", dest="hooks", action="store",
+        default=defaults.get("hooks", "{}"),
+        help="the scripts which get executed when events occure such has \
```

Comment:
Typos:

occure -> occur
such has -> such as

Also I don't think the `\` is necessary because Python has implied line continuation inside parentheses, so you can do:

    parser.add_argument(...
        help="the scripts which get executed when events occur such as "
             "saving notes (default: %(default)s)")